tests/tests_unit/test_base.py

```diff
@@ -139,8 +139,8 @@ def test_dump_camel_case(self):
 
     def test_load(self):
         assert MyResource(1).dump() == MyResource._load({"varA": 1}).dump()
-        assert MyResource(1, 2).dump() == MyResource._load({"var_a": 1, "var_b": 2}).dump()
-        assert {"var_a": 1} == MyResource._load({"var_a": 1, "var_c": 1}).dump()
```

Comment:
bug in the test? Strange that this passed at all.

Reply:
The old code would try every single key, while mine only grabs the specific camelCased keys we expect to load, hence the old also accepted snake_case (which was a serious bug, passing `asset_id=1` and `assetId=2` would not raise an error but give varying results depending on how it was serialized/deserialized).